feats/feats.yml

```diff
@@ -23,28 +27,37 @@
     As a focus action, you may change between any two forms (including your primary form or any alternate form). You maintain this capability in all of your forms.
     Each form is treated as a completely different character for mechanical purposes - possessing different attributes, feats, perks, flaws, and other defining characteristics. Your alternate form does, however, retain the ability to transform back into your primary form.
     In order to keep track of hit points, you should always record the total damage that your character has suffered. When transforming, your damage remains with you even if your maximum hit points change. For example, Dr. Jekyll has a max HP of 15 and Mr. Hyde has a max HP of 30. During combat, Mr. Hyde suffers 10 damage. When he later transforms back into Dr. Jekyll, the 10 damage remains and is subtracted from his new maximum, leaving the doctor with 5 out of 15 hit points.
-
   special: |
     When selecting feats for your alternate form, you may not select the <strong>Alternate Form</strong> feat.
     You may take this feat multiple times. Each time grants you access to an additional form.
 - !
   name: Area Manipulation (I - V)
   prerequisites:
     tier1:
-      Other:
-      - None
+      all:
+        Other:
+          all:
+          - None
     tier2:
-      Other:
-      - None
+      all:
+        Other:
+          all:
+          - None
```

Comment:
@jcte02 I don't know what I want to do here. I dislike the verbosity but when we look at implicit keys it just adds so much complexity to how it would be consumed by an end client. The more verbose syntax would keep our code clean when we process it on a given client